src/Core/src/Response.php

```diff
@@ -157,6 +158,12 @@ public function getStatusCode(): int
 
     public function toStream(): ResultStream
     {
+        $this->resolve();
```

Comment:
Im okey with resolving this first. It could throw exceptions. 

We should add the exceptions to the doc block.

Reply:
indeed. Fixed